wee_slack.py

```diff
@@ -3026,6 +3102,12 @@ def unfurl_ref(ref, ignore_alt_text, auto_link_display):
                 display_text = "#{}".format(ref.split('|')[1])
             elif id.startswith("@U"):
                 display_text = ref.split('|')[1]
+            elif id.startswith("!subteam"):
+                if ref.split('|')[1].startswith('@'):
+                    handle = ref.split('|')[1][1:]
+                else:
+                    handle = ref.split('|')[1]
```

Comment:
Hm, so the handle in `<!subteam^U2147483697|@group>` doesn't include `@` for you?

Reply:
In a channel I found some of the usergroups printing the `@` symbol twice like `@@dev`. I have only seen this in old
messages. These two lines account for a possible extra `@` sign and removes them. Maybe it just me.

Reply:
Ah, the web-client includes @ in the alt-text when creating a message, while wee-slack does not. I changed linkify_text to include @ in wee-slack as well.